js/components/common/react_modal.js.jsx

```diff
@@ -979,4 +979,43 @@ function convertTimeToString(time) {
   return ((time === 12.5 ? 12 : Math.floor(time)) % 12) + ":30" + meridiem
 }
 
-export { CourseModal, MapModal, FocusModal }
+/**
+ * For use in Generate - a warning modal for when the user enters one or more invalid courses
```

Comment:
should be amended to include the case that the user doesn't input any courses at all; but you can say something more generic like "user's course selection is invalid" or something